core/src/impl/Kokkos_Atomic_Exchange.hpp

```diff
@@ -378,32 +382,105 @@ inline void atomic_assign(volatile T* const dest, const T val) {
 #elif defined(KOKKOS_ENABLE_SERIAL_ATOMICS)
 
 template <typename T>
-inline T atomic_exchange(volatile T* const dest_v, const T val) {
-  T* dest  = const_cast<T*>(dest_v);
+inline T atomic_exchange(volatile T *const dest_v, const T val) {
+  T *dest  = const_cast<T *>(dest_v);
   T retval = *dest;
   *dest    = val;
   return retval;
 }
 
 template <typename T>
-inline void atomic_assign(volatile T* const dest_v, const T val) {
-  T* dest = const_cast<T*>(dest_v);
+inline void atomic_assign(volatile T *const dest_v, const T val) {
+  T *dest = const_cast<T *>(dest_v);
   *dest   = val;
```

Comment:
Why did all the above stuff change? I.e. all the spaces between * or & and the type or variable?

Reply:
We need to run the indenting script...